src/Cardano/Wallet/Binary.hs

```diff
@@ -600,3 +633,14 @@ decodeListIndef decodeOne = do
 blake2b256 :: forall tag. CBOR.Encoding -> Hash tag
 blake2b256 =
     Hash . BA.convert . hash @_ @Blake2b_256 . CBOR.toStrictByteString
+
+
+encodeList :: (a -> CBOR.Encoding) -> [a] -> CBOR.Encoding
+encodeList encodeOne list = mempty
+    <> CBOR.encodeListLen (fromIntegral $ length list)
+    <> mconcat (map encodeOne list)
+
+sign :: Tx -> [Witness] -> SignedTx
+sign tx witnesses = signedTxFromBS . toBS $ encodeSignedTx (tx, witnesses)
```

Comment:
Unsure about the location of this function as well as `SignedTx`

Reply:
(It is currently removed) but I don't like polluting the integration tests with imports to `CBOR` and `BL` + and forced to add `cborg` as dependency.

I do think/suspect something with the signature `:: Tx -> [Witness] -> SignedTx` should exist.